editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -2744,6 +2782,25 @@ void SpatialEditorViewport::_toggle_camera_preview(bool p_activate) {
 	}
 }
 
+void SpatialEditorViewport::_toggle_cinema_preview(bool p_activate) {
```

Comment:
To be a proper toggle, this method should actually be responsible for enabling the preview if `p_activate` is true, as is it's a bit confusing IMO.